REQUIRE

```diff
@@ -2,3 +2,4 @@ julia 0.5
 LearnBase
 StatsBase
 Iterators
+DataStructures
```

Comment:
Do we know a minimal version we need here? The people at METADATA always point this aspect out